exercises/practice/sieve/test_sieve.R

```diff
@@ -37,4 +37,11 @@ test_that("find primes up to 1000", {
                  967, 971, 977, 983, 991, 997))
 })
 
+test_that("division not used", {
+  division_ops <- c("/", "%", "sqrt", "\\^", "exp")
+  division_ops <- paste(division_ops, collapse = "|")
+  use_division <- any(grepl(division_ops, withVisible(sieve)$value))
```

Comment:
This doesn't work because `withVisible(sieve)$value` returns a function (closure) and not a string.

I don't see why `withVisible` is necessary though, surely one could simply use `deparse(sieve)`?

Reply:
You're right! Sorry, I must have mixed things up while trying to remove the dependency to `utils` by replacing `capture.output`. `deparse` will perfectly do the job. Thanks!